examples/advanced/edge/jobs/configs/cifar10_sync_lcp_config.json

```diff
@@ -0,0 +1,20 @@
+{
+ "endpoint": "http://localhost:4321",
+  "num_devices": 16,
+  "num_active_devices": 16,
+  "num_workers": 16,
```

Comment:
just a general question no need to solve in this PR.

this config is for the device simulator only and has nothing to do with a job.

we need to think where to put this.


V.S. for TPRunner case we do need these configs...